conf/scripts/create_index.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+
+# This script will create an ElasticSearch index using passed in command line
+# arguments for the host, port, index name and path to JSON file which includes
+# the index definition.
+
+SCRIPT_NAME=$0
+REQUIRED_NUM_ARGS=4
+
+# Fail fast if we get any errors
+set -e
+
+function usage {
```

Comment:
I know you took this from my example (my bad) but `function` is non-standard. Use `usage()` instead.